app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  # Associations
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  # Attributes
+  attribute :name, :string
+  attribute :bio, :text
+  attribute :posts_counter, :integer, default: 0
+  attribute :photo, :string
+
+  # Methods
+  def self.three_most_recent_posts(user)
+    user.posts.order(created_at: :desc).limit(3)
```

Comment:
[OPTIONAL] There was no need to get pass user as a parameter. Rails knows that `posts.order..` means `User.posts.order...`.

Reply:
Addressed